### PR TITLE
Add python bind for compute_aspect_ratio_of_cells

### PR DESCRIPTION
**Pull request by agrayver**
This is very useful for the mesh design.

Blocked by #9428 

Part of #9015

### CODE REVIEW

**Comment by agrayver** [comment]
Hm, first two commits are not supposed to be part of this PR, but should come from the #9428 

Is this because I didn't rebase this branch?

**Comment by peterrum** [comment]
FYI @nfehn

**Comment by masterleinad** [comment]
> Hm, first two commits are not supposed to be part of this PR, but should come from the #9428
> 
> Is this because I didn't rebase this branch?

You are targeting `master` so all commits in between will be part of this pull request. After #9428 is merged you might need to rebase (if there are any changes to these two commits).

**Comment by agrayver** [comment]
This was rebased and should be ready now.

File: contrib/python-bindings/include/triangulation_wrapper.h [hunk 1]
```diff
@@ -415,6 +416,14 @@ namespace python
     double
     maximal_cell_diameter() const;
 
+    /**
+     * Computes an aspect ratio measure for all active cells and fills a vector
+     * with one entry per cell.
+     */
+    boost::python::list
+    compute_aspect_ratio_of_cells(MappingQGenericWrapper mapping,
+                                  QuadratureWrapper      quadrature);
```
Comment (masterleinad) [inline contrib/python-bindings/include/triangulation_wrapper.h]: Is there any reason not to pass by `const&` here?

File: contrib/python-bindings/source/triangulation_wrapper.cc [hunk 2]
```diff
@@ -558,6 +558,34 @@ namespace python
     }
 
 
+    template <int dim, int spacedim>
+    boost::python::list
+    compute_aspect_ratio_of_cells(MappingQGenericWrapper &mapping_wrapper,
+                                  QuadratureWrapper &     quadrature_wrapper,
+                                  TriangulationWrapper &  triangulation_wrapper)
```
Comment (masterleinad) [inline contrib/python-bindings/source/triangulation_wrapper.cc]: and here?

**Comment by agrayver** [comment]
@masterleinad I made them const.

**Pull request merged**